src/app/components/Home/Services/Services.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import {
+  Badge,
+  Group,
+  Title,
+  Text,
+  Card,
+  SimpleGrid,
+  Container,
+  rem,
+} from "@mantine/core";
+import { IconGauge, IconUser, IconCookie } from "@tabler/icons-react";
+import classes from "@/app/components/Home/Services/Services.module.css";
+
+const mockdata = [
+  {
+    title: "Extreme performance",
+    description:
+      "This dust is actually a powerful poison that will even make a pro wrestler sick, Regice cloaks itself with frigid air of -328 degrees Fahrenheit",
+    icon: IconGauge,
+  },
+  {
+    title: "Privacy focused",
+    description:
+      "People say it can run at the same speed as lightning striking, Its icy body is so cold, it will not melt even if it is immersed in magma",
+    icon: IconUser,
+  },
+  {
+    title: "No third parties",
+    description:
+      "They’re popular, but they’re rare. Trainers who show them off recklessly may be targeted by thieves",
+    icon: IconCookie,
+  },
+];
```

Comment:
I think it is possible to define an `interface` to handle this one, to make it more strongly typed

```suggestion
interface ServicesContent {
    title: string,
    description: string,
    icon: react.ForwardRefExoticComponent<IconProps & react.RefAttributes<Icon>>,
}

const mockdata: ServicesContent[] = [
    {
        title: "Extreme performance",
        description:
            "This dust is actually a powerful poison that will even make a pro wrestler sick, Regice cloaks itself with frigid air of -328 degrees Fahrenheit",
        icon: IconGauge,
    },
    {
        title: "Privacy focused",
        description:
            "People say it can run at the same speed as lightning striking, Its icy body is so cold, it will not melt even if it is immersed in magma",
        icon: IconUser,
    },
    {
        title: "No third parties",
        description:
            "They’re popular, but they’re rare. Trainers who show them off recklessly may be targeted by thieves",
        icon: IconCookie,
    },
];
```

Reply:
PS: Don't just commit the suggestion, you can try implementing it yourself. but this will work, ;)  @LeaHaelin